fastlane/Pluginfile

```diff
@@ -6,6 +6,7 @@ group :screenshots, optional: true do
   gem 'rmagick', '~> 3.2.0'
 end
 
+ # TODO: Update to the next major version after: https://github.com/wordpress-mobile/release-toolkit/pull/319
```

Comment:
Off topic. I just realized the usage of the word "major". I assume that's intentional? As in we'll bump to 3.0 because going from hardcoded `develop` to hardcoded `trunk` is a breaking change.

Reply:
We initially intended to bump to 3.0, but then decided to go for a non-breaking approach for the moment. I'll shortly remove this TODO.